python/requirements.txt

```diff
@@ -1,4 +1,6 @@
-Flask==2.0
+Flask==2.0.0
```

Comment:
interesting... people typically like odd numbers better because they tend to be after a bugfix is released. ie:
2.0.0 -initial release
2.0.1 - bugfix
2.1.0 - new feature
2.1.1 - bug fix
... etc